### ISSUE TITLE
Add github webhooks

### ISSUE THREAD

**Issue by MarshallAsch** [SoorajModi/Botomir#77]
Sometimes you may want to get a notification in your discord server when someone creates a new Pull request or issue or when there is a new application release. And we can have that too!!

I know that there are bots that exist that do that, but since the goal of Botomir is to encompass as much as we can, why not add this?

This is probably something that should be configured through the settings web UI rather than through a settings command. 

You should be able to select a web hook event type, a repository, and a branch to have this message be sent for, as well as configure the message, channel, and guild it gets sent to that. (perhaps we can use a handlebars template for some sample message format and pass in the webhook event object to the message renderer to be able to take variables directly from the event.

**Comment by MarshallAsch** [comment]
This is blocked by #79
